.github/actions/e2e_benchmark/action.yml

```
@@ -166,11 +168,12 @@ runs:
 
     - name: Upload benchmark results to action run
       if: (!env.ACT)
-      uses: actions/upload-artifact@a8a3f3ad30e3422c9c7b888a15615d19a852ae32 # v3.1.3
+      uses: ./.github/actions/artifact_upload
       with:
-        path: |
+        path: >
```

```suggestion
        path: |
```
assume a typo? :smile: